portal/patient/fwk/libs/verysimple/Authentication/PassPhrase.php

```diff
@@ -14,96 +12,60 @@
 class PassPhrase
 {
     /**
-     * *
-     * GetRandomPassPhrase returns a prounoucable password of the given length
-     * filtering out potential offensive words
-     * * @public
-     * * @param int $$pass_length
-     * * @return string $password.
+     * GetRandomPassPhrase returns a prounoucable password
+     * of the given length filtering out potential offensive words
+     *
+     * @public
+     * @param int $length
+     * @return string
      */
-    static function GetRandomPassPhrase($length = 8)
+    static function GetRandomPassPhrase(int $length = 8): string
     {
-        srand((double) microtime() * 1000000);
+        $randomizer = new \Random\Randomizer();
```

Comment:
@bradymiller ok, since you pointed it out, I rewrote this function. Is it used, though? :joy:

Reply:
yep, not used. but now very shiny and new :)